omrs-database-polling-repository-event-mapper/build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+// Artifact names are taken from the directory by default, set in settings.gradle to override
+// The 'name' for the maven artifact, and description are set here
+ext.name = 'Abstract database polling event mapper connector'
```

Comment:
The name will be used as the gradle artifact. I think we should set this to the desired short name - and figure out if you can follow the existing naming scheme (names are starting to get quite long)

The gradle publish could override this name, but I don't see a need for both?

Reply:
the other connectors seem to use natural names - are you suggesting a shortened hyphenated name for this?